.gitignore

```diff
@@ -41,6 +41,10 @@
 !/config/jwt/*-test.pem
 ###< lexik/jwt-authentication-bundle ###
 
+###> sylius/sylius ###
+/config/encryption/dev.key
+###< sylius/sylius ###
```

Comment:
It would be better to change the value in `.env` to `/config/encryption/test.key` as it's already there by default and the innards of these files don't really matter as long as it's not prod.